app/assets/javascripts/admin/addon/components/admin-config-areas/emojis-list.gjs

```diff
@@ -0,0 +1,92 @@
+import Component from "@glimmer/component";
+import { fn } from "@ember/helper";
+import { service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import i18n from "discourse-common/helpers/i18n";
+import ComboBox from "select-kit/components/combo-box";
+
+export default class AdminConfigAreasEmojisList extends Component {
+  @service dialog;
+  @service adminEmojis;
+
+  get filteredEmojis() {
+    return this.adminEmojis.filteredEmojis;
+  }
+
+  get sortedEmojis() {
+    return this.adminEmojis.sortedEmojis;
+  }
+
+  get emojiGroups() {
+    return this.adminEmojis.emojiGroups;
+  }
+
+  get sortingGroups() {
+    return this.adminEmojis.sortingGroups;
+  }
+
+  <template>
+    <div class="form-horizontal">
+      <div class="inline-form">
+        <ComboBox
+          @value={{this.adminEmojis.filter}}
+          @content={{this.sortingGroups}}
+          @nameProperty={{null}}
+          @valueProperty={{null}}
+        />
+      </div>
+    </div>
+
+    {{#if this.sortedEmojis}}
```

Comment:
We need an empty list placeholder here per the guidelines:

![image](https://github.com/user-attachments/assets/8f370474-536a-4cc0-9d4f-82acae626137)

I realised we are missing this in some other pages though, me included. Gamification is the only place that has it so far:

![image](https://github.com/user-attachments/assets/b6199b17-dbff-43fc-93dc-8fec510a13f3)

https://github.com/discourse/discourse-gamification/blob/77fd25b9756d3ff684a41a1bdd1e7ffba6c3342a/admin/assets/javascripts/discourse/templates/admin-plugins/show/discourse-gamification-leaderboards/index.hbs#L78-L85

Will make a quick component for us to use

Reply:
Here we go when this PR is merged you can use it https://github.com/discourse/discourse/pull/29816

Reply:
Works like a charm! 👌👨‍🍳